src/migrate.py

```diff
@@ -94,26 +108,30 @@ def rollback_migration(self):
                 (name, app)
                 VALUES (:name, :app)
             """),
-                name=f"{self.migration_file}.sql",
+                name=f"{self.migration_name}.sql",
                 app="app_rollback"
             )
         logging.info("ROLLBACK EXECUTED SUCCESSFULLY")
         exit(0)
 
+
 def main():
     parser = argparse.ArgumentParser(description='Migrate and rollback database scripts')
     parser.add_argument('command', help="command to execute inside dbms")
     parser.add_argument('--driver', help="SQL Driver to use")
-    parser.add_argument('--dbstring', help="Add dbstring to connection if you didn't set DATABASE_MIGRATION_URL environment var")
+    parser.add_argument('--dbstring', help="Add dbstring to connection if you didn't set DATABASE_URL environment var")
+    parser.add_argument('--migration_name', help="Inform migration name to create or rollback sql migation file")
+    
     args = parser.parse_args()
 
     load_dotenv()
 
-    if not args.dbstring and 'DATABASE_MIGRATION_URL' not in os.environ:
-        logging.error('dbstring is missing, you have to provide it as DATABASE_MIGRATION_URL environment variable, or via --dbstring positional argument of command')
+    if not args.dbstring and 'DATABASE_URL' not in os.environ:
```

Comment:
Environment variables are outdated with our environment, did you get the latest version of code with the last changes we have made?

Reply:
I thought so, because I cloned it. Maybe I've overwritten it, I'll revise it as soon as possible.